content/updates/2020-07/1-rules-UI.md

```diff
@@ -0,0 +1,21 @@
+---
+pim_announcement_img: ./img/rules_UI.png
+pim_announcement_alt_img: "The UI to manage the rules"
+pim_announcement_audience:
+- EE
+---
+
+# A brand new UI to manage your rules
```

Comment:
```suggestion
# New! Create and update rules from the PIM!
```
I'm trying the "!" here ;)

Reply:
Il manque les meta data non ?

Reply:
@morganen si toutes les actions/opérations ne sont pas encore dispo il faut peut-être le dire ?

Reply:
J'hésitais justement, je me demandais si ça ne faisait pas trop "négatif" de le dire ici, sachant que je le dis déjà dans la doc à plein de reprises. Mais oui c'est sûrement utile de le redire ici

Reply:
meta data rajoutées, sorry pour l'oubli.